target_pos: [0.5, 0.5, 0.5]
feedback_topic_name: /move_to_action/feedback
subscriber_topic: 